contracts/tests/conftest.py

```diff
@@ -33,6 +33,12 @@ def number_of_auction_participants():
     return number_of_participants
 
 
+@pytest.fixture(scope="session")
+def auction_address(accounts):
+    return accounts[4]
+    # Come said I should use 4. We should fix that index stuff here
```

Comment:
Then fix it before merging please.

Reply:
It's a different issue, I fail to see the need to fix it as part of this  PR.

Reply:
Then potentially remove the comment and create a github issue about cleaning this. The comment is not helping. (but really, I didn't believe it would be too much work to fix it, so creating an issue seems overkill.)

Reply:
a46e2749a89b62a3bdf8a289f88d1faec059